mystyle.css

```diff
@@ -303,15 +311,11 @@ form li {
   width: 100%;
 }
 
-.longbutton {
-  padding: 0 70px 0 70px;
-}
-
 .about-section {
   width: 100%;
   height: 100vh;
```

Comment:
- [ ] The about section does not like the original one but don't worry I will help you to fix it 😅. Lets remove ` height: 100vh;` and add `padding: 64px 0;`. **TIP 📌**: In most cases do not add height because it will be calculated automatically you can use padding to make it bigger if you want. In 1% situations, it can help 🙌⭐

Reply:
Done